app/src/main/java/com/github/orkest/ViewModel/auth/AuthViewModel.kt

```diff
@@ -79,30 +81,8 @@ open class AuthViewModel: ViewModel() {
 
         // Computes the path to store the user in : user/user-firstLetter/users
         // user-firstletter is a document containing a subcollection which contains the users's documents
-        val firstLetter = username.value.text[0].uppercase()
```

Comment:
I had to modify this function by adding error handling in my PR #77, hit me up If you have trouble merging it